amethyst_core/src/transform/components/local_transform.rs

```diff
@@ -136,20 +137,18 @@ impl LocalTransform {
     /// Set the position.
     pub fn set_position(&mut self, position: Point3<f32>) -> &mut Self {
         self.translation = position.into();
-        self.flag(true);
 
         self
     }
 
     /// Set the rotation using Euler x, y, z.
-    pub fn set_rotation<D: Into<Deg<f32>>>(&mut self, x: D, y: D, z: D) -> &mut Self {
-        self.rotation = Quaternion::from::<f32>(Euler {
-            x: x,
-            y: y,
-            z: z,
-        }).into();
-
-        self.flag(true);
+    pub fn set_rotation<D: Into<Deg<f32>> + Angle>(&mut self, x: D, y: D, z: D) -> &mut Self {
+        let rotation =
```

Comment:
Here, you can use `Quaternion::from(Euler {})`, should be safer.

Reply:
The problem is that the new version of cgmath changed traits on the Euler struct (to ` Angle`), leading to unimplemented trait errors for conversion between cgmath types

Reply:
Ugh. Ok.